.github/workflows/_create-alpha-release-tag.yml

```diff
@@ -33,13 +39,13 @@ jobs:
           fetch-depth: '0'
       - name: Tag release
         run: |
-          LATEST_RELEASE_NUMBER=$(git describe --abbrev=0 --tags --match "alpha_release-*" | awk -F- '{print $2}' || true)
+          LATEST_RELEASE_NUMBER=$(git describe --abbrev=0 --tags --match "${{ inputs.tag_prefix }}alpha_release-*" | awk -F- '{print $2}' || true)
```

Comment:
What might be a literal value of `${{ inputs.tag_prefix }}` here? Looking at https://payments-platform.atlassian.net/browse/PP-9303, what repos would this apply to (i.e. stubs, endtoend, pay-scripts)?

Reply:
for pay-scripts `proxy_` so it would become `proxy_alpha_release-XXX`

Reply:
Right now, pay-dockerfiles, pay-scripts, pay-ci potentially